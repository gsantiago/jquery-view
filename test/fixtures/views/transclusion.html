<div class="transclusion-test">
  <p>This is a transclusion test</p>
  <strong>{{this.props.label}}</strong>
  <div class="transclusion-here" :transclude></div>
</div>