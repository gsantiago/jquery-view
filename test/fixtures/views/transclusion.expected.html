<my-element label="Title"><div class="transclusion-test">
  <p>This is a transclusion test</p>
  <strong>Title</strong>
  <div class="transclusion-here">Please <span>Luke</span>, transclude me!</div>
</div></my-element>